static/js/subs.js

```diff
@@ -235,12 +235,13 @@ exports.add_sub_to_table = function (sub) {
         row_for_stream_id(sub.stream_id).click();
         stream_create.reset_created_stream();
     }
+    stream_ui_updates.update_check_button_for_sub(sub);
```

Comment:
This line doesn't look like a pure extraction; this one should go in its own commit.

Reply:
I removed the line, as `add_sub_table` function renders the check-button above, so don't require call of `update_check_button_for_sub` here.